WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -191,11 +193,13 @@ public void onFocusChange(View v, boolean hasFocus) {
             populateUsernameButtons(usernames);
         }
 
+        final boolean isPrivateSite = isPrivateSite();
```

Comment:
This variable is unnecessary since the click listener is the only place it's used.